src/registrar/utility/email.py

```diff
@@ -35,6 +36,10 @@ def send_templated_email(
 
     Raises EmailSendingError if SES client could not be accessed
     """
+    if switch_is_active("disable_email_sending") and not settings.IS_PRODUCTION:
+        message = "Could not send email. Email sending is disabled due to switch 'disable_email_sending'."
+        raise EmailSendingError(message)
```

Comment:
Why are we raising an error here? we should just no-op here after logging that email sending is turned off, right?